src/front/DataInspector.js

```diff
@@ -2,7 +2,18 @@ L.TileLayer.XRay = L.TileLayer.extend({
 
     getTileUrl: function (tilePoint) {
         this.options.version = Date.now();
-        this.options.showLayer = L.K.Config.dataInspectorLayer || '';
+        var keys = Object.keys(L.K.Config);
+        if (L.K.Config['dataInspectorLayer___all__'] === true) {
```

Comment:
@mapmeld you just need to give nested path when creating fields, using dot as separator:

```
['dataInspector.layerName', {handler: L.FormBuilder.LabeledCheckBox, text: 'Show…
```